lib/tasks/sanitize_data.rake

```diff
@@ -0,0 +1,162 @@
+    end
+
+    def sanitize_payment_method(payment_method)
+      payment_method.name = "Payment Method #{payment_method.id}"
+      payment_method.description = payment_method.name
+      payment_method.save!
+    end
+
+    def sanitize_credit_card(card)
+      card.month = card.year = card.start_month = card.start_year = 1
+      card.cc_type = card.first_name = card.last_name = card.last_digits = "Dummy"
+      card.save!
+    end
+
+    def sanitize_payment(payment)
+      payment.response_code = payment.avs_response = payment.cvv_response_code = nil
+      payment.identifier = payment.cvv_response_message = nil
+      payment.save!
+    end
+
+    def sanitize_paypal_express_checkout(paypal_express_checkout)
+      paypal_express_checkout.token = nil
+      paypal_express_checkout.save!
+    end
+
+    def sanitize_stripe_account(stripe_account)
+      stripe_account.stripe_user_id = stripe_account.stripe_publishable_key = nil
+      stripe_account.save!
+    end
+
+    def sanitize_batch(model_class, sanitize_method_name, canned_users = nil)
+      model_class.constantize.all.each_with_index do |object, index|
+        sanitized_user = sanitized_user(canned_users, index) if canned_users
+
+        if sanitized_user
+          puts "Sanitizing #{model_class} #{object.id} with #{sanitized_user[:first_name]}"
+          __send__ sanitize_method_name, object, sanitized_user
+        else
+          puts "Sanitizing #{model_class} #{object.id}"
+          __send__ sanitize_method_name, object
+        end
+      end
+    end
+
+    def sanitized_user(canned_users, index)
+      sanitized_user = canned_users[index % canned_users.size]
+      sanitized_user[:email] = "#{index}_#{sanitized_user[:email]}"
+      sanitized_user
+    end
+
+    def build_canned_users
+      base_canned_users.each do |user|
+        user[:email] = "ofn@example.com"
+        user[:phone] = "0123456789"
+      end
+    end
+
+    def base_canned_users
+      [{ first_name: "Bob", last_name: "Jones" },
+       { first_name: "Cindy", last_name: "Rest" },
+       { first_name: "Pete", last_name: "Smith" },
+       { first_name: "Tony", last_name: "Ballantyne" },
+       { first_name: "Ben", last_name: "Raven" },
+       { first_name: "Robyn", last_name: "Monster" },
+       { first_name: "Nako", last_name: "Tolkein" },
+       { first_name: "Helen", last_name: "Mitcham" },
+       { first_name: "Emma", last_name: "Low" },
+       { first_name: "Mandy", last_name: "Trust" }]
+    end
+
+    # We need to disable user coNFirmation emails that would otherwise be triggered
+    def disable_user_confirmation_emails
+      Spree::UserMailer.class_eval do
+        def confirmation_instructions(*) end
+      end
+    end
+
+    def guard_and_warn
+      if Rails.env.production?
+        puts "This task cannot be executed in production"
+        exit
+      end
+
+      message = "This will permanently change DB contents, are you sure you want to proceed? (y/N)"
```

Comment:
but will `N` actually be the default? I thought you had to use `Question#default` for that

Reply:
there was no default, I have added no as default now.